setup.py

```diff
@@ -43,6 +43,7 @@
     classifiers=[
         "License :: OSI Approved :: BSD License",
         "Operating System :: OS Independent",
+        "Programming Language :: Python :: 2.6",
```

Comment:
We dropped Python 2.6 support with `datadogpy==0.14.0`. As a result, our CI does not run anymore tests using this version of Python. 
Let's not add it as a classifier as we don't guarantee that things work, and will in the future.